proposals/stage-1/PROJECT_ONBOARDING/PROJECT_ONBOARDING.md

```diff
@@ -0,0 +1,22 @@
+# Onboarding Checklist
+
+This is an informational checklist to help projects onboard into the OpenJS Foundation. If you have any questions or need help, the OpenJS Foundation CPC is available to assist.
+
+- [ ] Adopt the OpenJS Foundation Code of Conduct
+- [ ] Update project CoC reporting methods to include OpenJS Foundation escalation path
+- [ ] Transfer official domains to OpenJS Foundation
```

Comment:
Similarly, this feels like putting the cart before the horse; i should forfeit ownership of assets prior to obtaining the membership which preserves my access to those assets?

Reply:
Definitely keep your ownership prior to becoming a member; as you onboard we'll figure out what timeframe makes sense to transition assets over once you've become an OpenJS Foundation project